test/dummy/config/application.rb

```diff
@@ -7,6 +7,7 @@
 require 'action_cable/engine'
 require 'rails/test_unit/railtie'
 require 'sprockets/railtie'
+require 'jasmine'
```

Comment:
Given the pain caused by this – is it worth adding a comment or more of a length description in commit?